src/quick-lint-js/error.h

```diff
@@ -1391,6 +1391,10 @@
       { source_code_span comment_close; },                                     \
       ERROR(QLJS_TRANSLATABLE("unopened block comment"), comment_close))       \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_unused_arrow_expression, "E0211", { source_code_span where; },     \
+      WARNING(QLJS_TRANSLATABLE("unused arrow expression"), where))            \
```

Comment:
Nit: I'd say "function" instead of "expression".
```suggestion
      WARNING(QLJS_TRANSLATABLE("unused arrow function"), where))            \
```